tune_sklearn/tune_search.py

```diff
@@ -645,18 +646,20 @@ def _tune_run(self, config, resources_per_trial):
             search_algo = ConcurrencyLimiter(
                 search_algo, max_concurrent=self.n_jobs)
 
-        analysis = tune.run(
-            trainable,
-            search_alg=search_algo,
-            scheduler=self.early_stopping,
-            reuse_actors=True,
-            verbose=self.verbose,
-            stop=stop_condition,
-            num_samples=self.num_samples,
-            config=config,
-            fail_fast=True,
-            resources_per_trial=resources_per_trial,
-            local_dir=os.path.expanduser(self.local_dir),
-            loggers=self.loggers)
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
```

Comment:
I think we should only catch the warning for this `fail_fast=raise` but not blanket ignore all warnings?

Reply:
done